ldi-orchestrator/ldio-connectors/ldio-http-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioHttpInProcess.java

```diff
@@ -4,11 +4,23 @@
 import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiAdapter;
 import be.vlaanderen.informatievlaanderen.ldes.ldio.types.LdioInput;
 import io.micrometer.observation.ObservationRegistry;
+import org.springframework.context.ApplicationEventPublisher;
 
 public class LdioHttpInProcess extends LdioInput {
 	public static final String NAME = "Ldio:HttpIn";
 
-	public LdioHttpInProcess(String pipelineName, ComponentExecutor executor, LdiAdapter adapter, ObservationRegistry observationRegistry) {
-		super(NAME, pipelineName, executor, adapter, observationRegistry);
+	public LdioHttpInProcess(String pipelineName, ComponentExecutor executor, LdiAdapter adapter,
+							 ObservationRegistry observationRegistry, ApplicationEventPublisher applicationEventPublisher) {
+		super(NAME, pipelineName, executor, adapter, observationRegistry, applicationEventPublisher);
+	}
+
+	@Override
+	protected void resume() {
+		//Handled by status check
+	}
+
+	@Override
+	protected void pause() {
+		//Handled by status check
 	}
```

Comment:
These implementations should not be empty but entail the pausing/resume logic.

the halted flag should be set here